app-misc/recoll/recoll-1.40.1.ebuild

```diff
@@ -60,9 +50,6 @@ DEPEND="
 
 BDEPEND="
 	qt6? ( dev-qt/qttools:6 )
-	!qt6? (
-		qt5? ( dev-qt/linguist-tools:5 )
-	)
```

Comment:
You will likely want to have `qttools[linguist]` if this was going to fulfill the purpose that `dev-qt/linguist-tools:5` had.